lib/rocket_chat/request_helper.rb

```diff
@@ -110,7 +110,7 @@ def create_request(path, options)
         req = Net::HTTP::Post.new(path, headers)
         add_body(req, body) if body
 
-        form_data = reject_nils(options[:form_data])
+        form_data = options[:form_data]
```

Comment:
this would appear to be a breaking change. Can you explain why you've made it?

Reply:
I don't think it will be a breaking change and here is why:
1. `reject_nils` expect hash, form data in this update will be array. 
2. `form_data` was added in my prev PR with file upload and wasn't supported previously in any requests. `reject_nils` for json-bodies are still at row 107
3. `form_data` is added in `lib/rocket_chat/messages/room.rb:322` by transforming provided hash to an array of arrays.
3.1. If some params passed by user are `nil` they will be cleaned out by `compact` at `lib/rocket_chat/messages/room.rb:350`
3.2. If file-related optional params (filename, content-type) are `nil` they will be cleared at `lib/rocket_chat/messages/room.rb:353`
3.3. I don't see any possible way to receive an array with `nil` element at `lib/rocket_chat/messages/room.rb:354` with 3.1. and 3.2.

On the other hand, for future development, part of `file_upload_array` at lines `lib/rocket_chat/messages/room.rb:352-358` could be moved to request helper to cover all form_data requests from any future endpoints. I could do this in separate PR if this is ok